### ISSUE TITLE
isbntools fail under Python 3.7

### ISSUE THREAD

**Issue by workflowsguy** [xlcnd/isbntools#100]
When running any of the isbntools commands from the command line with Python 3.7 installed, an error message like this is displayed:

```
Traceback (most recent call last):
  File "/usr/local/bin/to_isbn13", line 7, in <module>
    from isbntools.bin.to_isbn13 import main
  File "/usr/local/lib/python3.7/site-packages/isbntools/bin/to_isbn13.py", line 5, in <module>
    from ..app import quiet_errors, to_isbn13
  File "/usr/local/lib/python3.7/site-packages/isbntools/app.py", line 16, in <module>
    from ._initapp import config, registry, CONF_PATH, CACHE_FILE
  File "/usr/local/lib/python3.7/site-packages/isbntools/_initapp.py", line 100, in <module>
    registry.setdefaultservice(v)
  File "/usr/local/lib/python3.7/site-packages/isbnlib/registry.py", line 23, in setdefaultservice
    services['default'] = services[name]
KeyError: 'merge'
```

**Comment by xlcnd** [comment]
Thanks. This is a bug!

In order to fix your installation create a folder `~\.isbntools` and a configuration file (inside this folder) called `isbntools.conf` with the following content:

[MISC]
REN_FORMAT={firstAuthorLastName}{year}_{title}_{isbn}
DEBUG=False

[SYS]
URLOPEN_TIMEOUT=10
THREADS_TIMEOUT=12

[SERVICES]
DEFAULT_SERVICE=goob
VIAS_MERGE=parallel

**Comment by xlcnd** [comment]
This issue was solved with release `isbntools-4.3.16`.

**Comment by workflowsguy** [comment]
Great, thanks!
